controllers/ArmFPIDController.java

```diff
@@ -0,0 +1,97 @@
+package org.frc5587.lib.controllers;
+
+import edu.wpi.first.wpilibj.controller.PIDController;
+import edu.wpi.first.wpilibj.smartdashboard.SendableBuilder;
+import edu.wpi.first.wpilibj.controller.ArmFeedforward;
+
+public class ArmFPIDController extends PIDController {
+    protected double f, p, i, d;
+    protected ArmFeedforward armFF;
+    private boolean fDisabled = false;
+
+    // params fpid are Feedforward and PID gains, param sCosVA is an array of the ks, kcos, kv, and ka used by an ArmFeedForward
+    // all values should come from robot characterization.
+    public ArmFPIDController(double f, double p, double i, double d, double[] sCosVA) {
+        super(p, i, d);
+        this.f = f;
+        this.p = p;
+        this.i = i;
+        this.d = d;
+        armFF = new ArmFeedforward(sCosVA[0], sCosVA[1], sCosVA[2], sCosVA[3]);
+    }
+
+    // you can also pass an ArmFeedforward instead of an array of the values needed to create one
+    public ArmFPIDController(double f, double p, double i, double d, ArmFeedforward armFF) {
+        super(p, i, d);
+        this.f = f;
+        this.p = p;
+        this.i = i;
+        this.d = d;
+        this.armFF = armFF;
+    }
+
+    // returns true if Feedforward is enabled
```

Comment:
the documentation here should follow java docs convention. you can find a link to that here: https://www.tutorialspoint.com/java/java_documentation.htm
this applies to all of the comments in this file